csrc/scheduler/mma_utils.h

```diff
@@ -195,6 +195,36 @@ class WarpMmaSwizzler {
   //! output and mma inputs are scheduled in scheduleOperandRead, which must be
   //! called before this function.
   static void scheduleLdMatrix(TensorView* tv, MmaOperand operand);
+
+  //! Function to schedule the swizzled TMA box.
+  //! This functions works on the assumption that the TMA box is 2D
+  //! and the inner-dimension is less or equal to the swizzle size.
+  //! This doesn't work for the swizzle none mode. For more details
+  //! refer to the figure doc/dev/tma/swizzle.svg
+  static void swizzleTMABox(
```

Comment:
Is this still used?

Reply:
Thanks, missed pushing a local commit - removed.